core/store/migrations/migration1597695690/migrate.go

```diff
@@ -0,0 +1,166 @@
+        CREATE INDEX idx_offchainreporting_oracle_specs_updated_at ON offchainreporting_oracle_specs USING BRIN (updated_at);
+
+        CREATE TABLE pipeline_specs (
+            id SERIAL PRIMARY KEY,
+            source_dot_dag TEXT NOT NULL,
+            created_at timestamptz NOT NULL
+        );
+
+        CREATE INDEX idx_pipeline_specs_created_at ON pipeline_specs USING BRIN (created_at);
+
+        CREATE TABLE pipeline_task_specs (
+            id SERIAL PRIMARY KEY,
+            pipeline_spec_id INT NOT NULL REFERENCES pipeline_specs (id),
+
+            task_type TEXT NOT NULL,
+            task_json jsonb NOT NULL,
+
+            successor_id INT REFERENCES pipeline_task_specs (id),
+
+            created_at timestamptz NOT NULL
+        );
+
+        CREATE INDEX idx_pipeline_task_specs_pipeline_spec_id ON pipeline_task_specs (pipeline_spec_id);
+        CREATE INDEX idx_pipeline_task_specs_successor_id ON pipeline_task_specs (successor_id);
+        CREATE INDEX idx_pipeline_task_specs_created_at ON pipeline_task_specs USING BRIN (created_at);
+
+        CREATE TABLE pipeline_runs (
+            id BIGSERIAL PRIMARY KEY,
+            pipeline_spec_id BIGINT NOT NULL REFERENCES pipeline_specs (id),
+            created_at timestamptz NOT NULL
+            -- NOTE: Could denormalize here with finished_at/output/error of last task_run if that proves necessary for performance
+        );
+
+        CREATE INDEX idx_pipeline_runs_pipeline_spec_id ON pipeline_runs (pipeline_spec_id);
+        CREATE INDEX idx_pipeline_runs_created_at ON pipeline_runs USING BRIN (created_at);
+
+        CREATE TABLE pipeline_task_runs (
+            id BIGSERIAL PRIMARY KEY,
+            pipeline_run_id BIGINT NOT NULL REFERENCES pipeline_runs (id),
+            output JSONB,
+            error TEXT,
+            pipeline_task_spec_id BIGINT NOT NULL REFERENCES pipeline_task_specs (id),
+            created_at timestamptz NOT NULL,
+            finished_at timestamptz,
+            CONSTRAINT chk_pipeline_task_run_fsm CHECK (
+                error IS NULL AND output IS NULL AND finished_at IS NULL
+                OR
+                error IS NULL AND output IS NOT NULL AND finished_at IS NOT NULL
+                OR
+                output IS NULL AND error IS NOT NULL AND finished_at IS NOT NULL
+            )
+        );
+
+        -- NOTE: This table is large and insert/update heavy so we must be efficient with indexes
+
+        CREATE INDEX idx_pipeline_task_runs ON pipeline_task_tuns USING BRIN (created_at);
+        -- CREATE INDEX idx_pipeline_task_runs_finished_at ON pipeline_task_tuns WHERE finished_at IS NOT NULL USING BRIN (finished_at);
+
+        -- This query is used in the runner to find unstarted task runs
+        -- CREATE INDEX idx_pipeline_task_runs_unfinished ON pipeline_task_runs (finished_at) WHERE finished_at IS NULL;
```

Comment:
Why is this commented?

Reply:
There's a syntax error, wanted to save it for you to look at